Updates for local testing of tile cache server

danscales opened the pull request:
I mainly updated the README instructions with a few things:
  - change gfw-staging to {bastion_server}
  - make it clear that PLANET_API_KEY must be set
  - mention you can get the password, keys, etc. from AWS secrets manager
  - give special instructions for Linux hosts

For linux hosts, the docker ports are only forwarded correctly to the host if you change the ports config
from just '8088:80' to '127.0.0.1:8088:80'

danscales committed:
Updates for local testing of tile cache server

I mainly updated the README instructions with a few things:
  - change gfw-staging to {bastion_server}
  - make it clear that PLANET_API_KEY must be set
  - mention you can get the password, keys, etc. from
    AWS secrets manager
  - give special instructions for Linux hosts

For linux hosts, the docker ports are forward correctly to the host if
you change the ports config from just '8088:80' to '127.0.0.1:8088:80'

solomon-negusse commented on README.md:
````diff
@@ -18,9 +20,31 @@ Raster and vector tile API for datasets in the [GFW Data API](https://github.com
+the docker and gotten hung accessing the database. You can then start the docker by
+hand via:
+
+```docker run -it -p127.0.0.1:8088:80 --entrypoint /bin/bash gfw-tile-cache_app:latest```
````
missing space here `-p127.0.0.1:8088:80`

danscales replied:
It works without the space as well, but putting in the space is more standard, so did that change.

solomon-negusse reviewed (approved):
Thanks!

solomon-negusse commented:
@danscales: It'd be good to add that for the ssh tunnel to work, the ip address should be whitelisted for port 22 access in the security groups. Thanks.

danscales committed:
Added some instructions on ssh tunnel

From Solomon's comments, added text that for the ssh tunnel to work,
your host's ip address should be whitelisted for port 22 in the
core-default security group.

danscales commented:
> @danscales: It'd be good to add that for the ssh tunnel to work, the ip address should be whitelisted for port 22 access in the security groups. Thanks.

Good suggestion, done!

danscales committed:
Merge branch 'dev' into local-testing
